buildSrc/src/main/groovy/nva.publication.api.java-conventions.gradle

```diff
@@ -81,15 +81,15 @@ jacocoTestCoverageVerification {
             limit {
                 counter = 'METHOD'
                 value = 'COVEREDRATIO'
-                minimum = 1.000
+                minimum = 0.900
```

Comment:
hmmmm

Reply:
Should I create temporary tests instead of commented ones?

Reply:
But I think our tech-lead should have the final call. 
I do not know the plan forward; for instance when the migration should occur and when the test should be put back in. Later today? in four weeks?